.agent/workflows/documentation.md

```
@@ -1,287 +1,115 @@
 ---
-description: Generate and update per-module README.md and RELEASE-NOTES.md for DRN.Framework.* packages — maintains documentation in sync with code
+description: Generate and update per-module README.md and RELEASE-NOTES.md from repository-profile documentation modules while preserving human-written content.
 ---
 
 > **Standalone documentation workflow**
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)